stock_pack_operation_quick_lot/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Stock Pack Operation Quick Lot
+==============================
+
+This module allows set pack operation Lot Name and End of Life Date directly
+on picking operations
+
+Usage
+=====
+
+To use this module, you need to:
+
```

Comment:
#. Activate "Track lots or serial numbers" in Inventory -> Configuration -> Setting ->  Lots and Serial Numbers 
#. Note: the product must have "Tracking | By Lots" in the inventory tab

Reply:
This is in configuration.

Reply:
Added in configuration